src/reducers/channelStores/verusid.js

```diff
@@ -29,13 +33,19 @@ export const channelStore_verusid = (state = {
     case SIGN_OUT_COMPLETE:
```

Comment:
It seems that serviceChannelOpen stays true on SIGN_OUT_COMPLETE, which is inconsistent with how the previous wyre service implementation worked, and might possibly lead to calls being attempted from a state where the user isn't signed in?